CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.0.3]
+### Added
+- Checking for game existence on the platform PatchKit

```

Comment:
It should be placed under Fixed category. Always look at the thing from the user perspective.

What this update introduces is fix for handling situation when app has been removed from PatchKit.

Reply:
done